client/app/components/Password/index.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react';
+import css from './Password.scss';
+
+export type State = {
+  password: false,
+};
+
+export class Password extends React.Component<State> {
+  constructor() {
+    super();
+    this.state = { password: false }; //Initially password is not seen.
+  }
+
+  showPassword = () => {
+      let password=document.getElementById("pw");
```

Comment:
We shouldn't be using vanilla JS since we're already using React.

Let's build this `InputPassword` component using the new Hooks API available in React. Here is [documentation](https://reactjs.org/docs/hooks-state.html) on it.

Let me know if you need any help. We'll also want to write automated tests for this component.